packages/@aws-cdk/aws-stepfunctions-tasks/lib/athena/start-query-execution.ts

```diff
@@ -101,14 +101,14 @@ export class AthenaStartQueryExecution extends sfn.TaskStateBase {
           's3:ListBucketMultipartUploads',
           's3:ListMultipartUploadParts',
           's3:PutObject'],
-        resources: [this.props.resultConfiguration?.outputLocation ?? '*'], // Need S3 location where data is stored https://docs.aws.amazon.com/athena/latest/ug/security-iam-athena.html
+        resources: ['*'], // State machines scoped to output location fail without * permissions  https://docs.aws.amazon.com/athena/latest/ug/security-iam-athena.html
```

Comment:
curious: is `*` needed on all of these APIs or does it apply to a subset of this list? if it's the latter, perhaps the policy statements should be broken up

Reply:
`s3:CreateBucket` was the only action that required wildcard permissions. This commit includes those changes 38cc8aa883a7b879fd3a7da708ffc21f1af5640c along with updates to the expected.json

Reply:
why not push a new policy statement with just createBucket scoped to `*`
i.e. - `policyStatements.push( new iam.PolicyStatement({...`

Reply:
This is out of date and the newest changes did that.